### DISCUSSION TITLE
Knobs ghosting/jittering on AKAI MPD218

### DISCUSSION THREAD

**Discussion by uio2pl** [rsjaffe/MIDI2LR#780] [Problems]
Hi Everyone! I experienced an issue with AKAI MPD218. All knobs have a small jittering. If I set the minimum value of 0, my controller is sending a frequent signal of 0 and 1 every half second. The same for example, if I set somthing between 65 and 66. It is sending 65 or 66 all the time. Midi2lr reads each signal, so it starts lagging and the lightroom screen is flashing because of small changes of exposure etc. Any ideas how to solve it?

**Comment by cindaaa** [comment]
hi

having the same issue, did you manage to find a solution?

**Comment by cindaaa** [comment]
OMGHIIIIII

i've had this problem for a good bit, but i just found a solution to it just tinkering, right after commenting here

thanks, bye!!!!!

.
.
.
.
.

(hehe, joking, here it is:)
-download the mpd218 editor
https://www.akaipro.com/downloads#mpd218
-load the preset from your device
-change anything
-send the preset back to the device
-change the setting back to the original (unless you wanted to make the change anyway, i guess) and send it to the device again
.
and that's it! at least it was for me, anyway. i also reinstalled the driver multiple times, unscrewed the nuts and removed the washers, did some good ol' percussive maintenance - but seeing as those did not seem to have any immediate effect, i don't think they played a role in the fix. still might be worth a shot, if the above steps do not seem to solve it.

**Reply by cindaaa** [reply]
oh, forgot to @uio2pl if the issue stands

but good luck to the fellow unfortunate souls in the future as well:D

**Comment by uio2pl** [comment]
I no longer have this issue. I'm not sure what solved it, but I recently re-enabled the knobs and noticed that the jittering stopped. I guess replacing the USB cable helped, since I had already made many changes with the MPD218 editor before, and I encountered this issue afterward.

**Comment by hellemmm** [comment]
Well.... people commented here have not mentioned true reason of the issue. They were just lucky at some point. The root problem is - potentiometers. They will not last forever. And you need to oil them time to time. The isue can be fixed without oil just turning them side-to-side. I use graphite oil for sliders and knobs - it works great. Also a soft pencil may help. Cleaning sliders surface with ethanol - may help also. Usually i do next: ethanol_cleaning -> soft_pencil_retouch -> praphite_oiling. Thats the perfect way to restore potentiomeners. If that wouldnt help - just replace them with new ones.
